### PR TITLE
add instructions for using db templates

### LINKED ISSUE #612: document DB template usage
need to document using the db templates and what they do.

@mnagy fyi.

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift/openshift-docs/issues/612

### CODE REVIEW

File: using_images/db_images/mongodb.adoc [hunk 1]
```diff
@@ -134,3 +134,11 @@ The MongoDB image can be run with mounted volumes to enable persistent storage f
 
 * *_/var/lib/mongodb_* - This is the database directory where
 MongoDB stores database files.
+
+== Creating a Database Service from a Template
+
+OpenShift provides a link:../../dev_guide/templates.html[template] to make creating a new database service easy.  The template provides parameter fields to define all the mandatory environment variables (user, password, database name, etc) with predefined defaults including auto-generation of password values.  It will also define both a link:../../dev_guide/deployments.html[DeploymentConfig] and a link:../../architecture/core_concepts/pods_and_services.html#services[Service].
+
+You can find instructions for instantiating templates by following this link:../../dev_guide/templates.html#creating-resources-from-a-template[instructions].
```
Comment (adellape) [inline using_images/db_images/mongodb.adoc]: s/this/these/

**Comment by bparees** [comment]
@adellape ok ready for review now.
@mnagy ptal.

plan would be to copy this content to the other 2 DB pages once approved.

File: using_images/db_images/mongodb.adoc [hunk 2]
```diff
@@ -134,3 +134,17 @@ The MongoDB image can be run with mounted volumes to enable persistent storage f
...
+
+The MongoDB templates should have been registered in the `openshift` project by the administrator during the link:../../admin_guide/install/first_steps.html[first steps] setup process.  There are two templates available: 
+
+* `mongodb-ephemeral` is for development/testing purposes only because it uses ephemeral storage for the database content.  This means that if the database Pod is restarted for any reason, such as the Pod being moved to another node or the DeploymentConfig being updated and triggered a redeploy, all data will be lost.
+* `mongodb-persistent` uses a persistent volume store for the database data which means the data will survive a Pod restart.  Using persistent volumes requires a persistent volume pool be defined in the OpenShift deployment.  Instructions for setting up the pool are located link:../../admin_guide/persistent_storage_nfs.html[here].
+
+
+You can find instructions for instantiating templates by following this link:../../dev_guide/templates.html#creating-resources-from-a-template[instructions].
```
Comment (adellape) [inline using_images/db_images/mongodb.adoc]: s/this/these/

**Comment by adellape** [comment]
Only the one comment, anything else would be minor formatting to do later.

**Comment by mnagy** [comment]
@bparees this is excellent. LGTM

**Comment by bparees** [comment]
thanks guys.  I'm going to copy this over to the other 2 DBs and update the PR.

**Comment by bparees** [comment]
copied. ready for final review/merge.

File: using_images/db_images/mongodb.adoc (hunk 2 above)
Comment (adellape) [inline using_images/db_images/mongodb.adoc]: s/this/these/ ! :neckbeard:

Comment (adellape) [inline using_images/db_images/mongodb.adoc]: And the other 2.

**Comment by bparees** [comment]
@adellape fixed.

**Comment by mnagy** [comment]
LGTM

**Pull request merged**